main.tf

```
@@ -0,0 +1,189 @@
+/**
+* ## Module: cloudposse/terraform-aws-lambda-elasticsearch-cleanup
+*
+* This module creates a scheduled Lambda function which will delete old
+* Elasticsearch indexes using SigV4Auth authentication. The lambda 
+* function can optionally send output to an SNS topic if the topic ARN
+* is given
+*/
+
+# Terraform
+#--------------------------------------------------------------
+terraform {
+  required_version = ">= 0.10.2"
+}
+
+# Data
+#--------------------------------------------------------------
+data "aws_iam_policy_document" "assume_role" {
+  statement {
+    actions = ["sts:AssumeRole"]
+    effect  = "Allow"
+
+    principals {
+      type        = "Service"
+      identifiers = ["lambda.amazonaws.com"]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "default" {
+  statement {
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+    ]
+
+    effect = "Allow"
+
+    resources = ["*"]
+  }
+
+  statement {
+    actions = [
+      "es:ESHttpGet",
+      "es:ESHttpPut",
+      "es:ESHttpPost",
+      "es:ESHttpHead",
+      "es:ESHttpDelete",
+      "es:Describe*",
+      "es:List*",
+    ]
+
+    effect = "Allow"
+
+    resources = [
+      "${var.es_domain_arn}",
+    ]
+  }
+}
+
+# Modules
+#--------------------------------------------------------------
+module "label" {
+  source     = "git::https://github.com/cloudposse/terraform-terraform-label.git?ref=tags/0.1.6"
+  namespace  = "${var.namespace}"
+  name       = "${var.name}"
+  stage      = "${var.stage}"
+  delimiter  = "${var.delimiter}"
+  attributes = "${compact(concat(var.attributes, list("elasticsearch", "cleanup")))}"
+  tags       = "${var.tags}"
+  enabled    = "true"
+}
+
+module "artifact" {
+  source      = "git::https://github.com/cloudposse/terraform-external-module-artifact.git?ref=tags/0.1.1"
+  filename    = "lambda.zip"
+  module_name = "terraform-aws-lambda-elasticsearch-cleanup"
+  module_path = "${path.module}"
+}
+
+# Locals
+#--------------------------------------------------------------
+locals {
+  function_name = "${module.label.id}-${module.label.attributes}"
```

```suggestion
  function_name = "${module.label.id}"
```

`attributes` are already added to the final `id` in the `label` module